store/src/java/com/zimbra/cs/ldap/unboundid/UBIDLdapFilterFactory.java

```diff
@@ -885,7 +885,9 @@ public ZLdapFilter accountsByCosesAndFeatureCheck(List<String> cosIds, String ld
     public ZLdapFilter accountsByCosAndFeatureCheck(String cosId, String ldapAttribute) {
         Cos cos = null;
         try {
-            cos = Provisioning.getInstance().get(Key.CosBy.id, cosId);
+            if(!cosId.contains("COS")) { // skipping dummy value cos fetching to avoid warning
```

Comment:
why this dummy check?

Reply:
Are we hard coding this value somewhere, its been there?

Reply:
We are passing dummy values when we are restarting our system. So during that time it is showing warning message "unable to fetch cos {COS_ID}". To prevent that we added this check so that we will fetch cos details if there is a valid cos Id.